v3/plugins/redhat/vscode-openshift-connector/0.1.1/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-openshift-connector
+version: 0.1.1
+type: VS Code extension
+displayName: OpenShift Connector
+title: OpenShift Connector
+description: Interacting with Red Hat OpenShift clusters and providing a streamlined developer experience using Eclipse Che
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins/redhat/vscode-openshift-connector
```

Comment:
Generally the repository should point at the repository that hosts the vscode plugin.

Reply:
updated